### ISSUE TITLE
DOC: How to reset nextstrain password? Get username if you don't remember

### ISSUE THREAD

**Issue by corneliusroemer** [nextstrain/cli#200]
I tried to use `nextstrain login` but couldn't figure out my password.

I'm lucky to remember the username as it's showing when I'm on nextstrain.org

I couldn't find a description on how to reset the password on the documentation page for `nextstrain login`.

I think this would be good to have documented. Even if there's no good way other than emailing.

https://docs.nextstrain.org/projects/cli/en/stable/commands/login/

@tsibley may know this

**Comment by corneliusroemer** [comment]
Also not clear: where does nextstrain cli look for AWS credentials so that it works without being logged in?

**Comment by tsibley** [comment]
Yes, this could be better documented and we could surface a "Reset your password" link on https://nextstrain.org/whoami.

To reset your password, go to nextstrain.org, click the _Login_ link in the top right (logout first if you're logged in on the web already, or use a private browsing mode window), click the _Forgot your password?_ link at the bottom of the login form, and follow the instructions from there.

**Comment by tsibley** [comment]
Doc on AWS credentials is at https://docs.nextstrain.org/projects/cli/en/stable/remotes/s3/.
